patches/react-mentions@4.4.7.patch

```diff
@@ -0,0 +1,797 @@
+-      suggestionsPosition: {}
++      suggestionsPosition: {},
++      setSelectionAfterHandlePaste: false
+     };
+     return _this;
+   }
+@@ -1894,6 +1931,13 @@ var MentionsInput = /*#__PURE__*/function (_React$Component) {
+         });
+         this.setSelection(this.state.selectionStart, this.state.selectionEnd);
+       }
++
++      if (this.state.setSelectionAfterHandlePaste) {
++        this.setState({
++          setSelectionAfterHandlePaste: false
++        });
++        this.setSelection(this.state.selectionStart, this.state.selectionEnd);
++      }
+     }
+   }, {
+     key: "componentWillUnmount",
+@@ -1935,7 +1979,7 @@ var MentionsInput = /*#__PURE__*/function (_React$Component) {
+       var newValue = spliceString(value, markupStartIndex, markupEndIndex, pastedMentions || pastedData).replace(/\r/g, '');
+       var newPlainTextValue = getPlainText(newValue, config);
+       var eventMock = {
+-        target: _objectSpread(_objectSpread({}, event.target), {}, {
++        target: _objectSpread$1(_objectSpread$1({}, event.target), {}, {
+           value: newValue
+         })
+       };
+@@ -1943,7 +1987,11 @@ var MentionsInput = /*#__PURE__*/function (_React$Component) {
+ 
+       var startOfMention = findStartOfMentionInPlainText(value, config, selectionStart);
+       var nextPos = (startOfMention || selectionStart) + getPlainText(pastedMentions || pastedData, config).length;
+-      this.setSelection(nextPos, nextPos);
++      this.setState({
++        selectionStart: nextPos,
++        selectionEnd: nextPos,
++        setSelectionAfterHandlePaste: true
++      });
+     }
+   }, {
+     key: "saveSelectionToClipboard",
+@@ -2005,7 +2053,7 @@ var MentionsInput = /*#__PURE__*/function (_React$Component) {
+       var newValue = [value.slice(0, markupStartIndex), value.slice(markupEndIndex)].join('');
+       var newPlainTextValue = getPlainText(newValue, config);
+       var eventMock = {
+-        target: _objectSpread(_objectSpread({}, event.target), {}, {
++        target: _objectSpread$1(_objectSpread$1({}, event.target), {}, {
+           value: newPlainTextValue
+         })
+       };
+@@ -2062,7 +2110,7 @@ var styled$3 = createDefaultStyle({
+     letterSpacing: 'inherit'
+   },
+   '&multiLine': {
+-    input: _objectSpread({
++    input: _objectSpread$1({
+       height: '100%',
+       bottom: 0,
+       overflow: 'hidden',
+diff --git a/dist/react-mentions.cjs.js b/dist/react-mentions.cjs.js
+index 168777643efee2afe58e34f9f147a4e246a4ba29..fd7a5910b8a4ff4e9156126ff1a1849adebfe5fa 100644
+--- a/dist/react-mentions.cjs.js
++++ b/dist/react-mentions.cjs.js
+@@ -1,7 +1,7 @@
+ 'use strict';
+ 
+ if (process.env.NODE_ENV === "production") {
+-  module.exports = require("./react-mentions.cjs.prod.js");
++  module.exports = require("./react-mentions-fix.cjs.prod.js");
+ } else {
+-  module.exports = require("./react-mentions.cjs.dev.js");
++  module.exports = require("./react-mentions-fix.cjs.dev.js");
```

Comment:
where is `react-mentions-fix`

Reply:
I copied my own built to the patch. Patch is hardly to edit directly.